package.json

```diff
@@ -25,6 +31,7 @@
     "ical.js": "1.2.x",
     "jquery": "2.2.x",
     "moment": "2.12.x",
+    "moment-timezone": "^0.5.7",
```

Comment:
The other dependencies have their major/minor versions constrained, so for consistency it may be worth doing the same here. Will defer to @sskhandek here though.